src/shared-components/toggle/index.tsx

```diff
@@ -0,0 +1,72 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+// @ts-ignore
+import ToggleButton from 'react-toggle-button';
+
+import { COLORS } from '../../constants';
+import {
+  Container,
+  Label,
+  ReactToggleButtonContainer,
+  trackStyle,
+  thumbStyle,
+} from './style';
+
+const propTypes = {
+  checked: PropTypes.bool,
+  label: PropTypes.string,
+  onChange: PropTypes.func.isRequired,
+};
+
+type ToggleProps = {
+  checked?: boolean;
+  label?: string;
+  onChange: () => void;
+};
+
+/**
+ * react-toggle-button in-lines all of its styling.
+ * toggleInputStyles overrides the defaults.
+ */
+const toggleInputStyles = {
+  margin: '0',
+  height: '24px',
+  width: '40px',
+  position: 'absolute',
+};
+
+const Toggle = ({ checked = false, label = '', onChange }: ToggleProps) => (
```

Comment:
`CHANGELOG`:

1. Update typing: make `onChange` required and rely on ES6 defaults instead of `defaultProps`. 
    - This required passing an `onChange` prop in the tests, too, which is the only changes there. 
1. Use `passThroughInputProps` option to override in-line styles on underlying `input` component. 
    - This was required to achieve the desired behavior via selectors in `ReactToggleButtonContainer`.